style.css

```diff
@@ -30,69 +19,82 @@ body > div {
 h1 {
     font-size: 24px;
     font-weight: 950;
-    color: #F9FAF8;
-    margin-right: auto;
+    color: #f9faf8;
+}
+
+.nav {
+    display: flex;
+    gap: 24px;
 }
 
 .heroMainText {
     font-size: 48px;
     font-weight: bolder;
-    color: #F9FAF8;
+    color: #f9faf8;
```

Comment:
Prettier is automatically formatting hex values to lowercase for purely aesthetic reasons.